hack/e2e/environment.sh

```diff
@@ -0,0 +1,19 @@
+# File used as metal3-dev-env config and as a source for variables used by envsubst
+
+# metal3-dev-env customization
+export IMAGE_OS=${IMAGE_OS:-"Ubuntu"}
+export CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-"docker"}
+export EPHEMERAL_CLUSTER=${EPHEMERAL_CLUSTER:-"kind"}
+export CAPI_VERSION=${CAPI_VERSION:-"v1alpha4"}
+export CAPM3_VERSION=${CAPM3_VERSION:-"v1alpha5"}
```

Comment:
why v1a4/v1a5? we are not there yet right? I think we can keep both `CAPI_VERSION` and `CAPM3_VERSION` set to `v1a3` && `v1a4` accordingly for now, and we will update it as part of [#167](https://github.com/metal3-io/cluster-api-provider-metal3/pull/167/files#diff-6cd24b3345ce830e47a9bb53e46046248df09d8cb27775f49cd514877ed78c17R4-R5)

Reply:
When running in the CI it's overridden by env vars, which [are already set to v1alpha4/alpha5](https://gerrit.nordix.org/gitweb?p=infra/cicd.git;a=blob;f=jjb/airship/job_capi_capm3_v1a5_e2e_tests_ubuntu.yml;h=04024720ff8a76b843fff6c4a9c191ae99c98ecd;hb=HEAD#l61). But I can stick to the old values as the defaults for manual runs.

Reply:
Yes, the point is as of now we do not have a working CAPM3 v1a5 with CAPI v1a4 yet, the vars can be set to anything, it just to avoid confusion for users thinking they are running e2e with latest CAPI.